Dockerfile

```diff
@@ -24,6 +24,7 @@ ARG RPM_LOCKFILE_PROTOTYPE_VERSION=0.9.0
 # specified in Renovate's package.json
 ARG NODEJS_VERSION=20.17.0
 
+ENV GOTOOLCHAIN=auto
```

Comment:
```suggestion
# Manager specific variables
ENV GOTOOLCHAIN=auto
ENV LANG=C.UTF-8
```

Can you please rebase this PR and group these options with some comment that they belong to specific managers?